pkg/storage/unified/search/bleve.go

```diff
@@ -36,7 +36,7 @@ type BleveOptions struct {
 	BatchSize int
 }
 
-type bleveBackend struct {
+type BleveBackend struct {
```

Comment:
keeping this private is helpfuil to avoid leaking the abstraction -- any reason it needs to be public?

Reply:
needed it in the metrics - fixed now! 👍